src/js/components/Paragraph/doc.js

```diff
@@ -29,3 +29,63 @@ export const doc = Paragraph => {
 
   return DocumentedParagraph;
 };
+
+export const themeDoc = {
+  'global.colors.text': {
+    description: 'The text color used inside the Paragraph.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: "{ dark: '#f8f8f8', light: '#444444' }",
+  },
+  'global.text': {
+    description: 'The text size of the Paragraph text.',
+    type: 'string',
+    defaultValue: 'medium',
+  },
+  'global.edgeSize.small': {
+    description: 'The padding around paragraph.',
+    type: 'string',
+    defaultValue: '12px',
+  },
+  'global.edgeSize': {
+    description: 'The possible sizes for gap, margin, and pad.',
+    type: 'object',
+    defaultValue: `{
+  edgeSize: {
+    none: '0px',
+    hair: '1px',
+    xxsmall: '3px',
+    xsmall: '6px',
+    small: '12px',
+    medium: '24px',
+    large: '48px',
+    xlarge: '96px',
+    responsiveBreakpoint: 'small',
+  },
+}`,
+  },
+  'paragraph.maxWidth': {
```

Comment:
I don't think we have a `paragraph.maxWidth`. We do have `paragraph.<size>.maxWidth`. I could see either adding that via `paragraph`, with an object showing each size, sort of like we have for `global.edgeSize` above. Or, we could add one per sub key, like `paragraph.<size>.maxWidth` and then show the maxWidth value for each size. I think the former would be a little more consistent, albeit a bit verbose.

Reply:
I adopted the former way, it is more consistent and readable imo